associates/kelly-thompson.json

```diff
@@ -0,0 +1,7 @@
+{
+	"first": "Kelly",
+	"last": "Thompson",
+	"twitter": "info_grrl"
```

Comment:
Oops, looks like there's a few commas missing here. Could you add those, commit them, and then push them to your branch please?